src/codeu/chat/util/CommandTokenizer.java

```diff
@@ -0,0 +1,68 @@
+package codeu.chat.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+final public class CommandTokenizer {
+
+	private String source;
```

Comment:
Should I use JavaDoc for these variables are just normal comments? I know JavaDoc is used by IDEs to provide method and variable description, but does that mean there is any disadvantage to using JavaDoc with private variables and methods?